### ISSUE TITLE
Phase 2: Move content-build Nightwatch Daily Accessibility Scan to GHA

### ISSUE THREAD

**Issue by pjhill** [department-of-veterans-affairs/va.gov-team#25568]
# Description
The `content-build` repository does not currently have a GitHub Actions workflow that executes the daily accessibility scan and reports the results to the #-daily-accessibility-scan channel in Slack. We need to move the daily scan out of Jenkins and into GHA.

## Acceptance Criteria
- [ ] A `content-build` GitHub Actions workflow executes the Nightwatch Accessibility Sitemap tests each morning starting at 8am ET.
- [ ] The results of the A11y Scan are reported to #-daily-accessibility-scan.
- [ ] Stretch: Improve the logging of errors

**Comment by timwright12** [comment]
Draft PR for this: 
https://github.com/department-of-veterans-affairs/content-build/pull/219

EOD status:
Tests are running. When they aren't in parallel Selenium seems to fail after about 5 mins, so I turned parallelization back on and they seem to be running smoothly (they just take forever to complete). After the current run is complete I'm going to try and manually trigger a failure to confirm they're running correctly. If all is well I'll then start working on the slack alert / output, then triggering on a schedule.

**Comment by timwright12** [comment]
Status:
Got the tests running and alerting slack, trying with a larger test run and need to make sure results are accurate.

**Comment by timwright12** [comment]
Still working on this, runs are taking over 4hrs and intermittently failing. Looking into the why

**Comment by timwright12** [comment]
Merged into master, will monitor cron. It should execute at 8am ET tomorrow. If it does, I'll update the slack channel and remove the Jenkins job

**Comment by timwright12** [comment]
Tests has stopped reporting failures in GHA and the scheduled job has been unable to start this morning. Need to investigate both, but overall the system doesn't seem stable.

**Comment by timwright12** [comment]
Daily tests in GHA have been running in parallel with Jenkins for a few days, reporting the same accessibility error. Waiting for on CMS team to resolve the a11y error to get a clean run, but all is looking good so far

**Comment by timwright12** [comment]
Pausing this work based on a [slack conversation](https://dsva.slack.com/archives/C01CHAY3ULR/p1625582297232700)

**Comment by 1Copenut** [comment]
7/7: Worked on the intermittent failing issue for static content in Cypress. Isolated the issue, but haven't determined why it's happening on some pages but not others.

**Comment by 1Copenut** [comment]
7/8: Consulted with Darius about some issues I was having with CORS errors. Verified I can recreate the issue, but not much closer to knowing if the CSS selector logic works properly.

**Comment by pjhill** [comment]
This ticket is OBE -- we are now pursuing Sitemap Accessibility scan using Cypress test framework in GHA instead.  This is not blocking the release of GHA as the merge-blocking CI tool for `vets-website`.
